### PR TITLE
added additional info field to questions

### PR DESCRIPTION
**Pull request by DavidSciola**
Ill do the tests later on

### CODE REVIEW

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java [hunk 1]
```diff
@@ -121,8 +122,9 @@ private void addResponse(Question q, String value) {
      * @return the HTML inputs for the question
      */
     private String getMultipleChoiceInputs(MultiplechoiceQuestion q) {
-        String s = "";
+        String s = "<hr>";
         s += "<h3> " + q.getPrompt() + " </h3>";
```
Comment (DavidHaighton) [inline src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:126]: https://github.com/wiverson/htmx-demo/blob/main/src/main/java/com/devhow/htmxdemo/InfiniteScroll.java#L47
You can make your inline html easier to read using an annotation. Heres an example.
Another way would be to use a thymeleaf block i think

**Comment by DavidHaighton** [comment]
you should include the tests in the same pr.

**Comment by DavidSciola** [comment]
> you should include the tests in the same pr.

Added the tests in this pr

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java (hunk 1 above)
Comment (DavidSciola) [reply src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:128]: Just added the inline html annotations

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java [hunk 2]
```diff
@@ -149,8 +155,10 @@ private String getOpenAnswerInput(OpenAnswerQuestion q) {
      * @return the HTML inputs for the question
      */
     private String getRangeInput(RangeQuestion q) {
-        String s = "";
+        @Language("html")
+        String s = "<hr>";
         s += "<h3>" + q.getPrompt() + "</h3>";
+        if (!Objects.equals(q.getAdditionalInfo(), "")) s += "additional notes: " + q.getAdditionalInfo() + "<br>";
```
Comment (kevinhquach) [inline src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:161]: again

File: src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java [hunk 3]
```diff
@@ -63,8 +64,10 @@ else if (q instanceof RangeQuestion) {
      * @return String of html elements
      */
     private String getMultipleChoiceResults(MultiplechoiceQuestion q) {
+        @Language("html")
         String s = "";
         s += "<h3>" + q.getPrompt() + "</h3>";
+        if (!Objects.equals(q.getAdditionalInfo(), "")) s += "additional notes: " + q.getAdditionalInfo() + "<br>";
```
Comment (kevinhquach) [inline src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java:70]: again

File: src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java [hunk 4]
```diff
@@ -95,9 +98,11 @@ private String getMultipleChoiceResults(MultiplechoiceQuestion q) {
      * @return String of html elements
      */
     private String getOpenAnswerResults(OpenAnswerQuestion q) {
+        @Language("html")
         String s = "";
         //doesn't quite work, but this should probably be done with css
         s += "<h3>" + q.getPrompt() + "</h3><div height='200px' overflow='scroll'>"; // could specify a class/id for the div
+        if (!Objects.equals(q.getAdditionalInfo(), "")) s += "additional notes: " + q.getAdditionalInfo() + "<br>";
```
Comment (kevinhquach) [inline src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java:105]: again

File: src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java [hunk 5]
```diff
@@ -117,8 +122,10 @@ private String getOpenAnswerResults(OpenAnswerQuestion q) {
      * @return String of html elements
      */
     private String getRangeResults(RangeQuestion q) {
+        @Language("html")
         String s = "";
         s += "<h3>" + q.getPrompt() + "</h3>";
+        if (!Objects.equals(q.getAdditionalInfo(), "")) s += "additional notes: " + q.getAdditionalInfo() + "<br>";
```
Comment (kevinhquach) [inline src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java:128]: again

**Review by kevinhquach — APPROVED** [review]
good, just small nitpicks

**Pull request merged**